### ISSUE TITLE
OpenIdConnect is used incorrectly

### ISSUE THREAD

**Issue by samedii** [HarryMWinters/fastapi-oidc#22]
This function does not take `issuer` as it's argument. It takes the full "well known uri".

```
oauth2_scheme = OpenIdConnect(openIdConnectUrl=issuer)
```
should be something like
```
oauth2_scheme = OpenIdConnect(openIdConnectUrl="http://localhost:8080/auth/realms/my-realm/.well-known/openid-configuration")
```

If I input this as the "issuer" to the config then everything is discovered and /docs with login works (but requests fail because the "issuer" doesn't match obviously)

I'll see if I can fork and fix this stuff

**Comment by samedii** [comment]
Added draft pull request #23

**Comment by HarryMWinters** [comment]
Hey @samedii! 

Thanks for bringing this up and double thanks for fixing it. The swagger doc UI isn't something I've looked at in a long time but it'll be great to have it working. I'll give some rough feedback on the PR tonight and I'll go through it in more detail over the next few days.

FWIW I believe #23 might also fix #19.
